resources/i18n/fr.json

```diff
@@ -174,7 +174,7 @@
                 "name": "Nom",
                 "streamUrl": "Lien du stream",
                 "homePageUrl": "Lien de la page d'accueil",
-                "updatedAt": "Mis à jour le",
+                "updatedAt": "Mise à jour le",
```

Comment:
When speaking of a radio _channel_, [radio](https://www.larousse.fr/dictionnaires/francais/radio/66020) is feminine.